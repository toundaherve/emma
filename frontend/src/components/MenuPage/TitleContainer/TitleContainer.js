import React from "react";

const TitleContainer = () => {
  return (
    <div className="row">
      <div className="col12">
        <div className="page-title menu-page-title-container">
          <h1>Appetizers</h1>
          <div
            className="breadcrumbs row"
            itemScope=""
            itemType="http://schema.org/BreadcrumbList"
          >
            <div className="col12">
              <span
                itemProp="itemListElement"
                itemScope=""
                itemType="http://schema.org/ListItem"
              >
                <span
                  itemScope=""
                  itemType="http://schema.org/Thing"
                  itemProp="item"
                >
                  <a href="/menu" itemProp="name" content="Menu">
                    Menu
                  </a>
                  <meta itemProp="url" content="/menu" />
                </span>
                <meta itemProp="position" content="1" />
              </span>
              <span className="arrow">›</span>
              <span
                itemProp="itemListElement"
                itemScope=""
                itemType="http://schema.org/ListItem"
              >
                <span
                  itemScope=""
                  itemType="http://schema.org/Thing"
                  itemProp="item"
                >
                  <strong itemProp="name">Appetizers</strong>
                  <meta itemProp="url" content="/menu/appetizers" />
                </span>
                <meta itemProp="position" content="2" />
              </span>
            </div>
          </div>
        </div>
        <div className="page-title menu-page-summary-container on-mobile-show">
          <div className="summary">
            All our appetizers are made to order and deliver big flavor that's
            simply too good not to share.
          </div>
        </div>
        <div className="page-title menu-page-summary-container on-mobile-show"></div>
        <div className="menu-dropdown-container">
          <div className="dropdown dropdown-menu-wrapper">
            <a
              id="menu-dropdown"
              className="btn small tertiary outline btn-dropdown-menu"
              href="#"
              data-toggle="dropdown"
            >
              <span>Menu</span>
              <span className="caret"></span>
            </a>
            <ul
              className="dropdown-menu dropdown-menu-right animated fadeIn"
              role="menu"
            >
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1041"
                  href="/menu/to-go-alcohol"
                >
                  To-Go Alcohol
                </a>
              </li>
              <li className="active">
                <a
                  id="menu-dropdown-go-to-category-C1000"
                  href="/menu/appetizers"
                >
                  Appetizers
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1004"
                  href="/menu/big-mouth-burgers"
                >
                  Big Mouth Burgers®
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1002"
                  href="/menu/ribs-steaks"
                >
                  Ribs &amp; Steaks
                </a>
              </li>
              <li>
                <a id="menu-dropdown-go-to-category-C1019" href="/menu/fajitas">
                  Fajitas
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1010"
                  href="/menu/salads-soups-chili"
                >
                  Salads, Soups &amp; Chili
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1005"
                  href="/menu/chicken-seafood"
                >
                  Chicken &amp; Seafood
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1001"
                  href="/menu/sandwiches"
                >
                  Sandwiches
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1025"
                  href="/menu/chicken-crispers"
                >
                  Chicken Crispers®
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1014"
                  href="/menu/guiltless-grill"
                >
                  Guiltless Grill
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1021"
                  href="/menu/tacos-quesadillas"
                >
                  Tacos &amp; Quesadillas
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1003"
                  href="/menu/smokehouse-combos"
                >
                  Smokehouse Combos
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1013"
                  href="/menu/lunch-specials"
                >
                  Lunch Specials
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1032"
                  href="/menu/3-for-10"
                >
                  3 for $10
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1027"
                  href="/menu/meal-for-2"
                >
                  Meal for 2
                </a>
              </li>
              <li>
                <a id="menu-dropdown-go-to-category-C1011" href="/menu/sides">
                  Sides
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1007"
                  href="/menu/desserts"
                >
                  Desserts
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1006"
                  href="/menu/beverages"
                >
                  Beverages
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1008"
                  href="/menu/kids-menu"
                >
                  Kids Menu
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1009"
                  href="/menu/party-platters"
                >
                  Party Platters
                </a>
              </li>
              <li>
                <a
                  id="menu-dropdown-go-to-category-C1026"
                  href="/menu/party-platter-add-ons"
                >
                  Party Platter Add Ons
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div className="page-title menu-page-summary-container on-mobile-hide">
          <div className="summary">
            All our appetizers are made to order and deliver big flavor that's
            simply too good not to share.
          </div>
        </div>
      </div>
    </div>
  );
};

export default TitleContainer;
