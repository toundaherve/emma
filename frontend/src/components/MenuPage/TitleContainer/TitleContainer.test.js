import React from "react";
import renderer from "react-test-renderer";

import TitleContainer from "./TitleContainer";

it("renders correctly", () => {
  const component = renderer.create(<TitleContainer />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
