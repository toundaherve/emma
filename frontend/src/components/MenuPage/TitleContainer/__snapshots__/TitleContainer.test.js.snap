// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col12"
  >
    <div
      className="page-title menu-page-title-container"
    >
      <h1>
        Appetizers
      </h1>
      <div
        className="breadcrumbs row"
        itemScope=""
        itemType="http://schema.org/BreadcrumbList"
      >
        <div
          className="col12"
        >
          <span
            itemProp="itemListElement"
            itemScope=""
            itemType="http://schema.org/ListItem"
          >
            <span
              itemProp="item"
              itemScope=""
              itemType="http://schema.org/Thing"
            >
              <a
                content="Menu"
                href="/menu"
                itemProp="name"
              >
                Menu
              </a>
              <meta
                content="/menu"
                itemProp="url"
              />
            </span>
            <meta
              content="1"
              itemProp="position"
            />
          </span>
          <span
            className="arrow"
          >
            ›
          </span>
          <span
            itemProp="itemListElement"
            itemScope=""
            itemType="http://schema.org/ListItem"
          >
            <span
              itemProp="item"
              itemScope=""
              itemType="http://schema.org/Thing"
            >
              <a
                content="Big Mouth Burgers®"
                href="/menu/big-mouth-burgers"
                itemProp="name"
              >
                Big Mouth Burgers®
              </a>
              <meta
                content="/menu/big-mouth-burgers"
                itemProp="url"
              />
            </span>
            <meta
              content="2"
              itemProp="position"
            />
          </span>
          <span
            className="arrow"
          >
            ›
          </span>
          <span
            itemProp="itemListElement"
            itemScope=""
            itemType="http://schema.org/ListItem"
          >
            <span
              itemProp="item"
              itemScope=""
              itemType="http://schema.org/Thing"
            >
              <strong
                itemProp="name"
              >
                Queso Burger*
              </strong>
              <meta
                content="/menu/big-mouth-burgers/queso-burger"
                itemProp="url"
              />
            </span>
            <meta
              content="3"
              itemProp="position"
            />
          </span>
        </div>
      </div>
    </div>
    <div
      className="page-title menu-page-summary-container on-mobile-show"
    >
      <div
        className="summary"
      >
        All our appetizers are made to order and deliver big flavor that's simply too good not to share.
      </div>
    </div>
    <div
      className="page-title menu-page-summary-container on-mobile-show"
    />
    <div
      className="menu-dropdown-container"
    >
      <div
        className="dropdown dropdown-menu-wrapper"
      >
        <a
          className="btn small tertiary outline btn-dropdown-menu"
          data-toggle="dropdown"
          href="#"
          id="menu-dropdown"
        >
          <span>
            Menu
          </span>
          <span
            className="caret"
          />
        </a>
        <ul
          className="dropdown-menu dropdown-menu-right animated fadeIn"
          role="menu"
        >
          <li>
            <a
              href="/menu/to-go-alcohol"
              id="menu-dropdown-go-to-category-C1041"
            >
              To-Go Alcohol
            </a>
          </li>
          <li
            className="active"
          >
            <a
              href="/menu/appetizers"
              id="menu-dropdown-go-to-category-C1000"
            >
              Appetizers
            </a>
          </li>
          <li>
            <a
              href="/menu/big-mouth-burgers"
              id="menu-dropdown-go-to-category-C1004"
            >
              Big Mouth Burgers®
            </a>
          </li>
          <li>
            <a
              href="/menu/ribs-steaks"
              id="menu-dropdown-go-to-category-C1002"
            >
              Ribs & Steaks
            </a>
          </li>
          <li>
            <a
              href="/menu/fajitas"
              id="menu-dropdown-go-to-category-C1019"
            >
              Fajitas
            </a>
          </li>
          <li>
            <a
              href="/menu/salads-soups-chili"
              id="menu-dropdown-go-to-category-C1010"
            >
              Salads, Soups & Chili
            </a>
          </li>
          <li>
            <a
              href="/menu/chicken-seafood"
              id="menu-dropdown-go-to-category-C1005"
            >
              Chicken & Seafood
            </a>
          </li>
          <li>
            <a
              href="/menu/sandwiches"
              id="menu-dropdown-go-to-category-C1001"
            >
              Sandwiches
            </a>
          </li>
          <li>
            <a
              href="/menu/chicken-crispers"
              id="menu-dropdown-go-to-category-C1025"
            >
              Chicken Crispers®
            </a>
          </li>
          <li>
            <a
              href="/menu/guiltless-grill"
              id="menu-dropdown-go-to-category-C1014"
            >
              Guiltless Grill
            </a>
          </li>
          <li>
            <a
              href="/menu/tacos-quesadillas"
              id="menu-dropdown-go-to-category-C1021"
            >
              Tacos & Quesadillas
            </a>
          </li>
          <li>
            <a
              href="/menu/smokehouse-combos"
              id="menu-dropdown-go-to-category-C1003"
            >
              Smokehouse Combos
            </a>
          </li>
          <li>
            <a
              href="/menu/lunch-specials"
              id="menu-dropdown-go-to-category-C1013"
            >
              Lunch Specials
            </a>
          </li>
          <li>
            <a
              href="/menu/3-for-10"
              id="menu-dropdown-go-to-category-C1032"
            >
              3 for $10
            </a>
          </li>
          <li>
            <a
              href="/menu/meal-for-2"
              id="menu-dropdown-go-to-category-C1027"
            >
              Meal for 2
            </a>
          </li>
          <li>
            <a
              href="/menu/sides"
              id="menu-dropdown-go-to-category-C1011"
            >
              Sides
            </a>
          </li>
          <li>
            <a
              href="/menu/desserts"
              id="menu-dropdown-go-to-category-C1007"
            >
              Desserts
            </a>
          </li>
          <li>
            <a
              href="/menu/beverages"
              id="menu-dropdown-go-to-category-C1006"
            >
              Beverages
            </a>
          </li>
          <li>
            <a
              href="/menu/kids-menu"
              id="menu-dropdown-go-to-category-C1008"
            >
              Kids Menu
            </a>
          </li>
          <li>
            <a
              href="/menu/party-platters"
              id="menu-dropdown-go-to-category-C1009"
            >
              Party Platters
            </a>
          </li>
          <li>
            <a
              href="/menu/party-platter-add-ons"
              id="menu-dropdown-go-to-category-C1026"
            >
              Party Platter Add Ons
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="page-title menu-page-summary-container on-mobile-hide"
    >
      <div
        className="summary"
      >
        All our appetizers are made to order and deliver big flavor that's simply too good not to share.
      </div>
    </div>
  </div>
</div>
`;
